mod/common.php

```diff
@@ -7,13 +7,12 @@
 use Friendica\Content\ContactSelector;
 use Friendica\Core\L10n;
 use Friendica\Database\DBA;
-use Friendica\Model\Contact;
-use Friendica\Model\GContact;
-use Friendica\Model\Profile;
+use Friendica\Model;
+use Friendica\Module\Contact;
```

Comment:
Although this is correct, please write `use Friendica\Module;` and later `Module\Contact::getTabsHTML` to prevent any immediate confusion.